packages/wonder-blocks-grid/components/row.js

```diff
@@ -116,7 +116,13 @@ class Row extends React.Component<{
 
         return (
             <View style={[styles.rowWrap, style]}>
-                <View style={[styles.row, hasMaxWidth && styles.rowMaxWidth]}>
+                <View
+                    style={[
+                        styles.row,
+                        maxWidth && styles.rowMaxWidth,
```

Comment:
🤷‍♂️ Minor nit: could you change these to be: `!!maxWidth && ...`? Right now it's potentially passing through `0` which I'm not sure we want, I think we want to enforce it to be a boolean.

Reply:
Oh, so I actually changed `maxWidth` to be a `number` (the width value itself, fixing your TODO) rather than a boolean. I think that means this is OK, but double-check for me?

Reply:
Yep - exactly! Sorry, to clarify: since `maxWidth` is a number now, if it fails we're going to be passing a number to the `style` prop. Namely, if `maxWidth` is `0` the results will be: `style={[styles.row, 0, ...]}`, which isn't what we want, we'd rather have `style={[styles.row, false, ...]}`. We can fix that by doing a simple `!!maxWidth` to cast the number to become a boolean. Not a big deal, just an edge case that we can fix.

Reply:
(If you do fix it you can just fix it before you land, no need for a re-review!)

Reply:
Oh, of course! Thank you for explaining; will do.